megamek/i18n/megamek/common/equipmentmessages.properties

```diff
@@ -616,6 +616,10 @@ SystemType.Cockpit.COCKPIT_INTERFACE=Interface\ Cockpit
 SystemType.Cockpit.COCKPIT_VRRP=Virtual\ Reality\ Piloting\ Pod
 SystemType.Cockpit.COCKPIT_QUADVEE=Quadvee\ Cockpit
 SystemType.Cockpit.COCKPIT_SUPERHEAVY_INDUSTRIAL=Superheavy\ Industrial\ Cockpit
+SystemType.Cockpit.COCKPIT_SUPERHEAVY_COMMAND_CONSOLE=Superheavy\ Command\ Console
+SystemType.Cockpit.COCKPIT_SMALL_COMMAND_CONSOLE=Small\ Command\ Console
+SystemType.Cockpit.COCKPIT_INDUSTRIAL_COMMAND_CONSOLE=Industrial\ Command\ Console
```

Comment:
Cockpit command consoles require advanced fire control. Industrial Mechs with advanced fire control use the standard cockpit type, so there is no reason to have a separate set of cockpit types for IMs with command console.